io.github.NickKarpowicz.LightwaveExplorer.json

```diff
@@ -0,0 +1,145 @@
+        "/lib/debug",
+        "/opt/intel"
+
+    ],
+    "modules": [
+        {
+            "name": "oneAPI",
+            "buildsystem": "simple",
+            "build-options":{
+                "append-path": "/usr/lib/sdk/llvm15/bin",
+                "prepend-ld-library-path": "/usr/lib/sdk/llvm15/lib"
+                },
+            "build-commands": ["chmod +x l_BaseKit_p_2023.2.0.49397_offline.sh",
+                            "./l_BaseKit_p_2023.2.0.49397_offline.sh -x -f webimage_extracted",
+                            "rm l_BaseKit_p_2023.2.0.49397_offline.sh",
+                            "webimage_extracted/l_BaseKit_p_2023.2.0.49397_offline/bootstrapper -s --action=install --components=intel.oneapi.lin.mkl.devel --install-dir=/app/opt/intel/oneapi --eula=accept",
+                            "rm -rf /app/opt/intel/oneapi/mkl/latest/lib/ia32",
+                            "rm -f /app/opt/intel/oneapi/mkl/latest/lib/intel64/*.so*",
+                            "rm -rf /app/opt/intel/oneapi/conda_channel"],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/992857b9-624c-45de-9701-f6445d845359/l_BaseKit_p_2023.2.0.49397_offline.sh",
+                    "sha256": "3dd04d2cc9ba2a8785580ba09068dbaf9aff9aa930b5fa7f10688210ea9be4a0"
+        
+                }
+            ]
+        },
+        {
+            "name": "fmt",
+            "config-opts": ["-DCMAKE_POSITION_INDEPENDENT_CODE=TRUE", "-DFMT_TEST=OFF"],
+            "buildsystem": "cmake-ninja",
+            "sources": [ 
+                { 
+                    "type": "git", 
+                    "url": "https://github.com/fmtlib/fmt", 
+                    "tag": "10.1.0",
+                    "commit": "e57ca2e3685b160617d3d95fcd9e789c4e06ca88" } 
+                ]
+        },
+        {
+            "name": "gcem",
+            "buildsystem": "cmake-ninja",
+            "sources": [ 
+                { 
+                    "type": "git", 
+                    "url": "https://github.com/kthohr/gcem", 
+                    "tag": "v1.17.0",
+                    "commit": "8422f5307b0498d09cf626f38acb03fe9f1efd94" } 
+                ]
+        },
+        {
+            "name": "dlib",
+            "buildsystem": "simple",
+            "build-commands":["cp -R dlib /app/include/"],
+            "sources": [ 
+                { 
+                    "type": "git", 
+                    "url": "https://github.com/davisking/dlib", 
+                    "tag": "v19.24.2",
+                    "commit": "39749fba5163d4e25030358f6aa8d91617bfdca2" } 
+                ]
+        },
+        {
+            "name": "cuda",
+            "buildsystem": "simple",
+            "build-commands":["mkdir /app/tmp","sh cuda_12.2.1_535.86.10_linux.run --silent --toolkitpath=/app/opt/cuda --toolkit --override --tmpdir=/app/tmp"],
```

Comment:
you still haven't explained how the aarch64 flatpak that is built will work (or even build)

Reply:
Is there a way to disable aarch64 entirely? The other dependency OneAPI is not compatible with it.

Reply:
I added an only-arches command to this effect, commit a46b776fa28f8f4821df3d1a02c37a681c206daf

Reply:
this won't do it

Reply:
https://docs.flathub.org/docs/for-app-authors/maintanance/#flathubjson

Reply:
Build succeeds after making flathub.json; thanks